### PR TITLE
[full-ci] prepare release, bump version

### PR DESCRIPTION
**Pull request by micbar**
## Description

- [x] Run Full CI
- [x] Tag
- [x] Merge back to master
- [ ] Create branch `stable-2.0`

## Related Issue

- Release ticket #5140 

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: changelog/2.0.0_2022-11-30/mount-ids.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Bugfix: Make storage users mount ids unique by default
+
+The mount ID of the storage users provider needs to be unique by default. We made this value configurable and added it to ocis init to be sure that we have a random uuid v4. This is important for federated instances.
+
+> **Warning**
+>BREAKING Change: In order to  make every ocis storage provider ID unique by default, we needed to use a random uuidv4 during ocis init. Existing installations need to set this value explicitly or ocis will terminate after the upgrade.
+> To upgrade from 2.0.0-rc.1 to 2.0.0-rc.2 you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```
Comment (kulmann) [inline changelog/2.0.0_2022-11-30/mount-ids.md:7]:
```suggestion
> To upgrade from 2.0.0-rc.1 to 2.0.0(-rc.2) you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```

or 

```suggestion
> To upgrade from 2.0.0-rc.1 to 2.0.0-rc.2 or 2.0.0 you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```

File: changelog/2.0.0_2022-11-30/mount-ids.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+>
+>You can also add
+>```
+>storage_users:
+>  mount_id: some-random-uuid
+>gateway:
+>  storage_registry:
+>    storage_users_mount_id: some-random-uuid
+>```
+>to the ocis.yaml file which was created during initialisazion
```
Comment (kulmann) [inline changelog/2.0.0_2022-11-30/mount-ids.md:17]:
```suggestion
>to the ocis.yaml file which was created during initialisation
```

File: changelog/2.0.0_2022-11-30/mount-ids.md (hunk 1 above)
Comment (phil-davis) [inline changelog/2.0.0_2022-11-30/mount-ids.md:7]:
```suggestion
> To upgrade from 2.0.0-rc.1 to 2.0.0-rc.2, 2.0.0 or later you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```

**Review by phil-davis — APPROVED** [review]
LGTM. I just made a small suggestion for the mount-ids.

**Review by kulmann — APPROVED** [review]
Some changelog nitpicks (see comments). Also, I've messed up the web changelog items a bit:
- `update-web-6.0.0-rc.6.md`,
- `update-web-6.0.0-rc.8.md` and
- `update-web-6.0.0.md`

contain all changes instead of the diff from one to the other. But I'd rather delete the two rc-version changelog files completely than switching to diffs, because I find the breaking change warnings rather important. What do you think? Delete `update-web-6.0.0-rc.6.md` and `update-web-6.0.0-rc.8.md`? Or keep it like it is?

The rest of the PR looks fine. Changelog folder name, changelog template (finally getting rid of the manually updated suffixes) and the version bumps all look good.

File: changelog/2.0.0_2022-11-30/mount-ids.md (hunk 1 above)
Comment (kulmann) [reply changelog/2.0.0_2022-11-30/mount-ids.md:7]: Yeah, had the same impression, we should not give the impression of needing to upgrade from rc.1 to rc.2 first and then to 2.0.0 final.

Comment (micbar) [reply changelog/2.0.0_2022-11-30/mount-ids.md:7]: thank you. Will add this.

**Comment by micbar** [comment]
> Some changelog nitpicks (see comments). Also, I've messed up the web changelog items a bit:
> 
> * `update-web-6.0.0-rc.6.md`,
> * `update-web-6.0.0-rc.8.md` and
> * `update-web-6.0.0.md`
> 
> contain all changes instead of the diff from one to the other. But I'd rather delete the two rc-version changelog files completely than switching to diffs, because I find the breaking change warnings rather important. What do you think? Delete `update-web-6.0.0-rc.6.md` and `update-web-6.0.0-rc.8.md`? Or keep it like it is?
> 
> The rest of the PR looks fine. Changelog folder name, changelog template (finally getting rid of the manually updated suffixes) and the version bumps all look good.

I will delete the two rc versions like you proposed.

File: changelog/2.0.0_2022-11-30/mount-ids.md (hunk 1 above)
Comment (micbar) [reply changelog/2.0.0_2022-11-30/mount-ids.md]: picked the first one.

File: changelog/2.0.0_2022-11-30/mount-ids.md (hunk 2 above)
Comment (micbar) [reply changelog/2.0.0_2022-11-30/mount-ids.md]: done

**Comment by micbar** [comment]
waiting for the final go / no go decision.

**Pull request merged**